tests/EventListener/ResizeFormListenerTest.php

```diff
@@ -15,16 +15,41 @@
 
 use PHPUnit\Framework\TestCase;
 use Sonata\Form\EventListener\ResizeFormListener;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\Form\DataMapperInterface;
 use Symfony\Component\Form\Exception\UnexpectedTypeException;
-use Symfony\Component\Form\Form;
+use Symfony\Component\Form\Extension\Core\DataMapper\DataMapper;
+use Symfony\Component\Form\Extension\Core\DataMapper\PropertyPathMapper;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilder;
 use Symfony\Component\Form\FormEvent;
 use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormFactoryBuilder;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
 
 /**
  * @author Ahmet Akbana <ahmetakbana@gmail.com>
  */
 final class ResizeFormListenerTest extends TestCase
 {
+    private FormFactoryInterface $factory;
+    private FormInterface $form;
+
+    /**
+     * @psalm-suppress UndefinedMethod
```

Comment:
Not sure why this is necessary, I'll try to take a look later

Reply:
Maybe it is about the chaining call for the form property?